Library/Login.cs

```diff
@@ -8,11 +8,12 @@ namespace Library
 {
     class Login
     {
+        LibraryDataEntities2 entities2 = new LibraryDataEntities2();
```

Comment:
Uffff... Faktas, kad susipainiosim, kai yra `LibraryDataEntities` ir `LibraryDataEntities2` :D
If it's possible - palik tik vieną